### PR TITLE
Added iOS notes about PWA and related

### PR DESCRIPTION
**Pull request by odahcam**
This is VERY important for those who wants to create a PWA with camera access.

Sources: 
- https://stackoverflow.com/questions/46228218/how-to-access-camera-on-ios11-home-screen-web-app
- https://github.com/webrtc/samples/issues/933

Implementation: https://minhahavan.com

### CODE REVIEW

File: features-json/stream.json [hunk 1]
```diff
@@ -277,8 +277,9 @@
       "9.3":"n",
       "10.0-10.2":"n",
       "10.3":"n",
-      "11.0-11.2":"y",
-      "11.3-11.4":"y",
+      "11.0-11.2":"y #3",
+      "11.3":"y",
```
Comment (HM100) [inline features-json/stream.json]: This one should be here. Please remove it

Comment (HM100) [reply features-json/stream.json]: Please do it

File: features-json/stream.json [hunk 2]
```diff
@@ -277,8 +277,9 @@
...
+      "11.4":"y #3",
```
Comment (HM100) [inline features-json/stream.json]: And change 11.4 to 11.3 -11.4
Per Repository rules, no new versions can be added or be splitted into other versions. Please fix that ASAP. Thank you for the time being!!!!

Comment (HM100) [reply features-json/stream.json]: Even if only affects 11.4 and not both 11.3 to 11.4, you mustn't break browser versions.

Comment (odahcam) [reply features-json/stream.json]: Damn, that can get very confusing, but ok.

**Comment by Fyrd** [comment]
Thanks!

**Comment by odahcam** [comment]
Yw!

**Comment by slokhorst** [comment]
Also doesn't seem to work in iOS 12. Can you confirm?

**Comment by odahcam** [comment]
Confirmed, at least `getUserMedia` does not work inside a standalone-running PWA. 😞

**Pull request merged**